docs/ingester-handover.md

```diff
@@ -0,0 +1,44 @@
+# Ingester Hand-over
+
+The [ingester](architecture.md#ingester) holds several hours of sample
+data in memory. When we want to shut down an ingester, either for
+software version update or to drain a node for maintenance, this data
+must not be discarded.
+
+Each ingester goes through different states in its lifecycle. When
+working normally, the state is `ACTIVE`.
+
+On start-up, an ingester first goes into state `PENDING`. After a
+short time, if nothing happens, it adds itself to the ring and goes
+into state ACTIVE.
+
+A running ingester is notified to shut down by Unix signal
```

Comment:
This paragraph should mention the `-ingester.claim-on-rollout` flag.

Reply:
#1566 to remove it